packages/sdk/src/types/runtime/actor.ts

```diff
@@ -80,20 +81,21 @@ export interface ActorSet {
  */
 export class Actor implements ActorLike, Patchable<ActorLike> {
     // tslint:disable:variable-name
-    private _internal: InternalActor;
+    private _internal = new InternalActor(this);

```

Comment:
I tested the use of `this` in in-line initializers, and it all works as if it were initialized from inside the constructor.